app/agents/voice/automatic/__init__.py

```
@@ -157,10 +159,27 @@ async def main():
 
+    smart_turn_analyzer = None
+    if config.ENABLE_FAL_SMART_TURN:
+        try:
+            session = create_aiohttp_session()
+            smart_turn_analyzer = FalSmartTurnAnalyzer(
+                aiohttp_session=session, api_key=config.FAL_SMART_TURN_API_KEY
+            )
+            logger.info(
+                "SMART_DEBUG: Fal.ai Smart Turn analyzer configured for transport-level integration"
+            )
+        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)